pkg/arvo/tests/sys/vane/eyre.hoon

```diff
@@ -996,7 +996,7 @@
                 %give
                 %response
                 %start
-                :-  200
+                :-  204
```

Comment:
This one shouldn't expect a `204` response, because there is data in there! Eyre is failing to meet this expectation, so I guess that's good.

Reply:
Weird. That shouldn't have slipped in there.